eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Epoch.hs

```diff
@@ -193,11 +173,13 @@ epochTransition = do
       TRC (epochState', UpecState pp (utxosPpups utxoSt'), ())
   let utxoSt'' = utxoSt' {utxosPpups = ppupSt'}
 
-  let Coin oblgCurr = obligation pp (rewards dstate') (psStakePoolParams pstate'')
-      Coin oblgNew = obligation pp' (rewards dstate') (psStakePoolParams pstate'')
+  let -- Coin oblgCurr = obligationDPState (DPState dstate' pstate'')
+      -- Now that obligation does not depend on the PParams at all, the current
+      -- and the new are identical, so we don't have to adjust for the difference.
+      Coin oblgNew = obligationDPState (DPState dstate' pstate'')
       Coin reserves = asReserves acnt'
       utxoSt''' = utxoSt'' {utxosDeposited = Coin oblgNew}
-      acnt'' = acnt' {asReserves = Coin $ reserves + oblgCurr - oblgNew}
+      acnt'' = acnt' {asReserves = Coin $ reserves}
```

Comment:
We could get rid of the `$` :  `Coin reserves`

Reply:
yes, good catch